kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/SelectedResources.vue

```diff
@@ -0,0 +1,183 @@
+                    @moveDown="() => {}"
+                  />
+                </DragHandle>
+                <div style="padding: 0 10px">
+                  <span style="flex: 1">
+                    <LearningActivityIcon
+                      :kind="lesson.learning_activities[0]"
+                      class="icon-style"
+                    />
+                  </span>
+                </div>
+                <div>
+                  <span class="arrange-item-block">
+                    <span>
+                      <KRouterLink
+                        v-if="lesson.link"
+                        :text="lesson.title"
+                        :to="lesson.link"
+                        style="font-size: 14px"
+                      />
+                    </span>
+                    <p style="font-size: 12px">{{ bytesForHumans(lesson.files[0].file_size) }}</p>
+                  </span>
+                </div>
+              </div>
+            </KGridItem>
+
+            <KGridItem
+              :layout12="{ span: 2 }"
+              :layout8="{ span: 3 }"
+              :layout4="{ span: 1 }"
+            >
+              <span class="add-minus-button">
+                <KIconButton
+                  icon="emptyTopic"
+                  @click="$emit('navigateToParent', lesson.id)"
+                />
+
+                <KIconButton
+                  icon="minus"
+                  @click="$emit('removeResource', lesson.id)"
+                />
+              </span>
+            </KGridItem>
+          </KGrid>
+        </Draggable>
+      </transition-group>
+    </DragContainer>
+  </div>
+
+</template>
+
+
+<script>
+
+  import DragSortWidget from 'kolibri-common/components/sortable/DragSortWidget';
+  import DragContainer from 'kolibri-common/components/sortable/DragContainer';
+  import DragHandle from 'kolibri-common/components/sortable/DragHandle';
+  import Draggable from 'kolibri-common/components/sortable/Draggable';
+  import commonCoreStrings from 'kolibri/uiText/commonCoreStrings';
+  import LearningActivityIcon from 'kolibri-common/components/ResourceDisplayAndSearch/LearningActivityIcon.vue';
+  import bytesForHumans from 'kolibri/uiText/bytesForHumans';
+  import { searchAndFilterStrings } from 'kolibri-common/strings/searchAndFilterStrings';
+
+  export default {
+    name: 'SelectedResources',
+    components: {
+      DragSortWidget,
+      DragContainer,
+      DragHandle,
+      Draggable,
+      LearningActivityIcon,
+    },
+    mixins: [commonCoreStrings],
+    setup() {
+      const { upLabel$, downLabel$ } = searchAndFilterStrings;
+      return {
+        upLabel$,
+        downLabel$,
+      };
+    },
+    props: {
+      resourceList: {
+        type: Array,
+        required: true,
+      },
+      currentLesson: {
+        type: Object,
+        required: true,
+      },
+      loading: {
+        type: Boolean,
+        required: true,
+      },
+    },
+
+    computed: {
```

Comment:
Could use the themeToken `fineLine` here https://design-system.learningequality.org/colors/#tokens-fineLine